packages/generator-electrode/generators/app/templates/gulpfile.js

```diff
@@ -1 +1,2 @@
+process.env.ES6_LINT = true;
```

Comment:
The task `lint-server-test` is using `.eslintrc-mocha-test` which is extending `es5-node` configurations. 
This flag will switch the new generated apps to use the `eslintrc-mocha-test-es6` config which extends `es6-node` config for the `lint-server-test` task.